.github/workflows/nightly-pipeline.yaml

```
@@ -104,6 +104,42 @@ jobs:
           propagate_failure: true
           trigger_workflow: true
           wait_workflow: true
+  cuopt-build:
+    needs: [get-run-info, rmm-build, dask-cuda-build]
```

```suggestion
    needs: [get-run-info, rmm-build, cudf-build, raft-build]
```

`cuopt` only depends on `rmm`, `cudf`, and `raft`, right?

if so this line should be updated according to this suggestion.

this will ensure that those dependencies build and publish before the `cuopt` build starts.